governance/overview.md

```diff
@@ -0,0 +1,60 @@
+ - **Enactment period** — time between a proposal being approved and enacted (make law). It is also the minimum period necessary to lock funds to propose an action
+ - **Lock period** — time (after the proposal enactment) that the tokens of the winning voters are locked. Users can still use this tokens for staking or voting
+ - **Delegation** — act of transfering your voting power to another account for up to a certain conviction
+
+## Principles
+
+Guiding "soft" principles for engagement with Moonbeam's governance process include:
+
+ - Being inclusive to token holders that want to engage with Moonbeam and that are affected by governance decisions.
+ - Favoring token holder engagement, even with views contrary to our own, versus a lack of engagement.
+ - A commitment to openness and transparency in the decision-making process.
+ - Working to keep the greater good of the network ahead of any personal gain.  
+ - Acting at all times as a moral agent that considers the consequences of action (or inaction) from a moral standpoint.
+ - Being patient and generous in our interactions with other token holders, but not tolerating abusive or destructive language, actions, and behavior.
+
+These points were heavily inspired by Vlad Zamfir’s writings on governance. Refer to his articles, [especially this one](https://medium.com/@Vlad_Zamfir/how-to-participate-in-blockchain-governance-in-good-faith-and-with-good-manners-bd4e16846434).
+
+## On-Chain Governance Mechanics
+
+The "hard" governance process for Moonbeam will be driven by an on-chain process and will leverage the Democracy, Council, and Treasury [Substrate frame pallets](/resources/glossary/#substrate-frame-pallets), similar to how Kusama and the Polkadot Relay Chain are governed. The overall intent of these modules are to allow the majority of tokens on the network to determine the outcomes of key decisions around the network. These decision points come in the form of stake-weighted voting on proposed referenda.
+
+Some of the main components of this governance model include:
+
+ - **Council** — A group of elected individuals who have special voting rights within the system. Council members are expected to propose referenda for voting and have an ability to veto publicly-sourced referenda. There are rolling elections for council members where GLMR holders will vote on new or existing council members.
+ - **Treasury** — A collection of funds that can be spent by submitting a proposal along with a deposit. Spending proposals must be approved by the council. Rejected proposals will result in the proposer losing their deposit.
+
+See [this overview on the Polkadot website](https://polkadot.network/a-walkthrough-of-polkadots-governance/) and [this wiki post](https://wiki.polkadot.network/docs/en/learn-governance) for more details on how these Substrate frame pallets implement on-chain governance.
```

Comment:
There is currently no council or treasury. Maybe this is to set expectations for the future?

Reply:
Yes you are right, but I've added a bullet point with Referendum and voting as well, it was sort of incomplete.